packages/keyring-controller/src/utils.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Throws an error.
+ *
+ * @param error - Error message or Error object to throw.
+ */
+export function throwError(error: string | Error): never {
```

Comment:
This seems like such a simple function that I'm curious if we need it, but if it was copied from somewhere else, then I suppose that's fine.

Reply:
We copied this one from `eth-keyring-controller`, but it is used in only three occurrences, and all of them just pass a string - so I think this is just adding cognitive overload.

removed in [a9712ca](https://github.com/MetaMask/core/pull/3830/commits/a9712caf6fc3e401563c2e1add1b10b753c06d40)